### DISCUSSION TITLE
doris自动分析问题

### DISCUSSION THREAD

**Discussion by Muzi999** [apache/doris#34708] [A - General / Q&A] [answered]
版本: doris2.1.2
FE警告日志中一直在执行 这样的SQL
SELECT SUBSTRING(CAST(MIN(key) AS STRING), 1, 1024) as min,
选择 SUBSTRING(CAST(MIN(key) AS STRING), 1, 1024) 作为最小值，
SUBSTRING(CAST(MAX(key) AS STRING), 1, 1024) as max
SUBSTRING(CAST(MAX(key) AS STRING), 1, 1024) 作为最大值
FROM DB.Table index null
FROM DB.Table 索引为空
然后每次执行都不成功
错误为
Encountered: INDEX 遇到：索引
Expected: COMMA, INDEX is keyword, maybe INDEX

**Comment by yagagagaga** [comment]
日志截图能看看吗

**Reply by Muzi999** [reply]
![image](https://github.com/apache/doris/assets/54492956/6499ab86-4c3e-414f-a4f5-0f50d33e2ce1)

**Reply by yagagagaga** [reply]
sql中查询的那张表的schema能看一下吗

**Reply by Muzi999** [reply]
![image](https://github.com/apache/doris/assets/54492956/5f1546a5-e037-4cad-9635-50eb6090a12e)

**Reply by Muzi999** [reply]
| TABLE_CATALOG | TABLE_SCHEMA | TABLE_NAME      | TABLE_TYPE | ENGINE | VERSION | ROW_FORMAT | TABLE_ROWS | AVG_ROW_LENGTH | DATA_LENGTH | MAX_DATA_LENGTH | INDEX_LENGTH | DATA_FREE | AUTO_INCREMENT | CREATE_TIME          | UPDATE_TIME          | CHECK_TIME | TABLE_COLLATION | CHECKSUM | CREATE_OPTIONS | TABLE_COMMENT |
|---------------|--------------|----------------|------------|--------|---------|------------|------------|----------------|-------------|-----------------|--------------|-----------|----------------|----------------------|----------------------|------------|-----------------|----------|----------------|---------------|
| internal      | CDP          | Tag_UserHistory| BASE TABLE | Doris  |         |            | 2265       | 21             | 49326       |                 |              |           |                | 2024-02-01 19:57:43.000 | 2024-03-28 10:41:11.000 |            | utf-8           |          |                | OLAP          |

**Comment by ixzc** [comment, marked as answer]
it has been fixed. you can upgrade to 2.1.3
https://github.com/apache/doris/pull/33423
